plugins/by-name/vim-suda/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, helpers, ... }:
+
+lib.nixvim.vim-plugin.mkVimPlugin {
+  name = "vim-suda";
+  globalPrefix = "suda#";
+  maintainers = [ lib.maintainers.marcel ];
+
+  settingsOptions = {
+    executable = helpers.defaultNullOpts.mkStr "sudo" ''
+      Path to the sudo binary.
+    '';
+
+    noninteractive = helpers.defaultNullOpts.mkBool false ''
```

I don't think these are _proper_ booleans, i.e. `set foo` and `set nofoo`. Based on the default `0` listed in the docs, I think these are bit flag style _fake_ booleans.

If so:

```suggestion
    noninteractive = helpers.defaultNullOpts.mkFlagInt 0 ''
```